test/hotspot/jtreg/serviceability/jvmti/vthread/VThreadEventTest/libVThreadEventTest.cpp

```diff
@@ -90,10 +95,10 @@ Agent_OnAttach(JavaVM *vm, char *options, void *reserved) {
   err = jvmti->SetEventNotificationMode(JVMTI_ENABLE, JVMTI_EVENT_VIRTUAL_THREAD_END, nullptr);
   check_jvmti_error(err, "SetEventNotificationMode for VirtualThreadEnd");
 
-  err = jvmti->SetEventNotificationMode(JVMTI_ENABLE, EXT_EVENT_VIRTUAL_THREAD_MOUNT, nullptr);
+  err = set_ext_event_notification_mode(jvmti, JVMTI_ENABLE, "VirtualThreadMount", NULL);
```

Comment:
For consistency in the test source, can we use `nullptr` instead of `NULL`?

Reply:
I considered that, but all other, existing calls to `set_ext_event_notification_mode()` use `NULL`.